assets/js/aplus.js

```diff
@@ -606,3 +606,31 @@ $.fn.highlightCode = function(options) {
         });
     };
 })(jQuery, window, document);
+
+
+/**
+ * prevent double submit of forms
+ */
+
+$(function() {
+
+    $.fn.preventDoubleSubmission = function() {
+  $(this).on('submit',function(e){
+    var $form = $(this);
+
+    if ($form.data('submitted') === true) {
+      // Previously submitted - don't submit again
+      e.preventDefault();
+    } else {
+      // Mark it so that the next submit can be ignored
+      $form.data('submitted', true);
```

Comment:
Disabling the submit button might be a bit better user experience. So something like `$form.find('[type="submit"]').prop('disabled', 'disabled')`.